### DISCUSSION TITLE
rdbc-mysql-0.1.6\src\lib.rs:253:38  error

### DISCUSSION THREAD

**Discussion by yk0n9** [rbatis/rbatis#243] [Q&A]
![图片](https://user-images.githubusercontent.com/66518156/187108823-64ad40f2-3c1d-4e80-af00-9ecdf0b134f8.png)

keyword: "".to_owned(),
    |             ^^^^^^^ expected enum `Keyword`, found struct `std::string::String`

**Comment by zhuxiujia** [comment]
> ![图片](https://user-images.githubusercontent.com/66518156/187108823-64ad40f2-3c1d-4e80-af00-9ecdf0b134f8.png)
> 
> keyword: "".to_owned(), | ^^^^^^^ expected enum `Keyword`, found struct `std::string::String`

Is it the latest version?
the last version is
```toml
rbdc-mysql = "0.1.15"
```
I recommend locking 0.1 and run command ``` cargo update```

```toml
rbdc-mysql = "0.1"
```

**Reply by yk0n9** [reply]
对

**Reply by zhuxiujia** [reply]
> 对

0.1.6 It's an outdated version

**Reply by yk0n9** [reply]
ok
